compiler/damlc/lib/DA/Cli/Damlc/Packaging.hs

```diff
@@ -217,6 +221,83 @@ disableScenarioService opts = opts
 toGhcModuleName :: LF.ModuleName -> GHC.ModuleName
 toGhcModuleName = GHC.mkModuleName . T.unpack . LF.moduleNameString
 
+installDataDep ::
+     Options
+  -> NormalizedFilePath
+  -> FilePath
+  -> [DecodedDalf]
+  -> MS.Map (UnitId, b) LF.DalfPackage
+  -> MS.Map UnitId LF.DalfPackage
+  -> [UnitId]
+  -> LF.PackageId
+  -> UnitId
+  -> LF.DalfPackage
+  -> IO ()
+installDataDep opts projectRoot dbPath pkgs stablePkgs dependenciesSoFar depUnitIds pkgId unitId dalfPackage = do
```

Comment:
So many arguments! Maybe worth passing a record here instead (and taking advantage of RecordWildCards)?

Reply:
good idea, I went ahead and did the same for `generateAndInstallIfaceFiles` and `buildLfPackageGraph`